pkg/querier/querierpb/querier.proto

```diff
@@ -60,8 +61,12 @@ message EvaluateQueryResponseScalarValue {
 
 message EvaluateQueryResponseInstantVectorSeriesData {
   int64 nodeIndex = 1;
-  repeated cortexpb.Sample floats = 2 [(gogoproto.nullable) = false];
-  repeated cortexpb.FloatHistogramPair histograms = 3 [(gogoproto.nullable) = false];
+  repeated InstantVectorSeriesData series = 2 [(gogoproto.nullable) = false];
```

Comment:
Is it safe to reuse the field 2? From what I understand this might break wire format compatibility. Unless if this proto is not using to communicate across query-frontend <-> querier.